deploy-olm-operator.yml

```diff
@@ -4,10 +4,9 @@
   become: false
   gather_facts: false
 
-  vars_files:
-    - "{{ playbook_dir }}/../../../config/ansible/vault.yml"
 
   vars:
+    vault_file_path: "{{ vault_file_path | default('{{ playbook_dir }}/../../../config/ansible/vault.yml')}}"
```

Comment:
@jsztuka Wrong syntax will cause errors in downstream it should be as follows: 
```
vault_file_path: "{{ vault_file_path | default(playbook_dir +'/../../../config/ansible/vault.yml') }}"
```

Reply:
@jsztuka  bump

Reply:
Thanks for notice!